roles/sympa/tasks/sympa_db_pg.yml

```diff
@@ -26,8 +26,8 @@
 - name: Create PostgreSQL user for Sympa
   postgresql_user:
     db: sympa
-    name: "{{ sympa_db_app_user }}"
-    password: "{{ sympa_db_app_password }}"
+    name: "{{ sympa.db.app_user }}"
```

Comment:
why no vault for postgres?

Reply:
Oh, because Postgres was added by @racke _before_ I added the vault.
This will be fixed.

Reply:
Fixed by commit 7946e1f7a66a739d412a06d06f18b00f9ddff8ad.